src/components/screen-renderer.vue

```diff
@@ -95,7 +95,7 @@ export default {
   },
   methods: {
     rebuildScreen(definition) {
-      if (!isEqual(definition, this.currentDefinition)) {
+      if (!isMatch(this.currentDefinition, omit(definition, "isMobile"))) {
```

Comment:
Please check if omit `isMobile` has a negative effect. This test is failing: https://cloud.cypress.io/projects/c4pb2f/runs/1699/test-results/cb552109-cccd-4b12-8be7-b57ec5b633da/replay?att=1&ts=1735855217053.788&utm_campaign=view+Test+Replay&utm_medium=failed&utm_source=github

Reply:
solved